postgres/values.yaml

```diff
@@ -13,5 +13,6 @@ password: DEFINED-IN-REGION-CHART
 port_public: '5432'
 port_private: '5432'
 port_metrics: '9113'
+hba_cidr: 100.0.0.0/8
```

Comment:
Its convenient for us to define this as a default. But others might have setup there k8s pod CIDR range differently so I wonder if we should maybe default to `0.0.0.0` and be more specific in  out regional charts. Just a thought. It would require us to specify the range for each component seperatly.